src/test/testUtil.ts

```diff
@@ -240,6 +240,25 @@ export async function assertTextEditorContains(contents: string): Promise<void |
     }
 }
 
+export async function assertTabSize(size: number): Promise<void | never> {
```

Comment:
"count" might be a more familiar name.


```suggestion
export async function assertTabCount(size: number): Promise<void | never> {
```